nahusenayElias/Team_4
# DRD-106: Add styles to Careers page

## Related ticket
[DRD-106](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-106)

## In this PR: 
- Added an About Druid section to Careers page, I added a About Us paragraph to the Basic page in Drupal. 
- Added a About title field to About us paragraph
- Added more content to the Jobs page in Drupal
- Increased hero image height
- Added buttons to direct to Druid's application and open jobs pages
- removed padding from Layout.jsx
- removed unnecessary icons
- Added fixed height to Services cards 
- Changed text alignment to left in Services single pages

## Testing instructions
- checkout branch
- lando drush cim
- lando db-import DRD-106.sql.gz
- check Jobs page that all content is rendering correctly 

[DRD-106]: https://edu-team4-react24k.atlassian.net/browse/DRD-106?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
